cmd/thanos/query.go

```diff
@@ -108,6 +108,8 @@ func registerQuery(app *extkingpin.App) {
 	maxConcurrentSelects := cmd.Flag("query.max-concurrent-select", "Maximum number of select requests made concurrently per a query.").
 		Default("4").Int()
 
+	queryMetricLabels := cmd.Flag("query.metric-label", "Optional selection of metrics to be collected").Strings()
```

Comment:
collected where?

Reply:
same with name - it's very vague now.

Reply:
Also what are possible values here? We really need `Enum` not `String` right?

Reply:
Yes Enum makes sense

Reply:
I think you should also add defaults here?

Reply:
yh I've done that now @metonymic-smokey